SpecFlowMarsProject/StepDefinitions/LanguagePageStepDefinitions.cs

```diff
@@ -0,0 +1,104 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using SpecFlowMarsProject.Pages;
+using SpecFlowMarsProject.Tests;
+using SpecFlowMarsProject.Utilities;
+using System;
+using System.Diagnostics;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowMarsProject.StepDefinitions
+{
+    [Binding]
+    public class LanguagePageStepDefinitions : CommonDriver
+    {
+        //IWebDriver driver;
+        //IWebDriver driver = new ChromeDriver();
+        LoginPage loginPageObj = new LoginPage();
+        LanguagePage languageObj = new LanguagePage();
+
+        [BeforeScenario]
+        public void SetUpTimeMaterial()
+        {
```

Comment:
this needs to be moved in to a hooks class all hooks should live there, unless its specific to the actual class logic